### PR TITLE
Rcx fix

### PR DESCRIPTION
**Pull request by rlutes**
Allow diagnostics to run if zone reheat data is not present.

### CODE REVIEW

File: pnnl/AirsideRCxAgent/airside/diagnostics/schedule_reset_aircx.py [hunk 1]
```diff
@@ -280,7 +280,7 @@ def unocc_fan_operation(self):
             _log.info(msg)
             diagnostic_msg = {"low": 60.0, "normal": 60.0, "high": 60.0}
 
-        if 64.2 not in diagnostic_msg.values():
+        if 64.2 not in list(diagnostic_msg.values()):
```
Comment (craigpnnl) [inline pnnl/AirsideRCxAgent/airside/diagnostics/schedule_reset_aircx.py:285]: what is 64.2?  I see a low high normal referenced on line 281 but where does the 64.2 come in?

File: pnnl/EconomizerRCxAgent/economizer/diagnostics/EconCorrectlyOn.py [hunk 2]
```diff
@@ -156,17 +181,8 @@ def economizer_on_algorithm(self, cooling_call, oat, rat, mat, oad, econ_conditi
...
-        if elapsed_time >= self.data_window and len(self.timestamp) >= self.no_required_data:
-            if elapsed_time > self.max_dx_time:
-                _log.info(constants.table_log_format(self.analysis_name, self.timestamp[-1], (constants.ECON2 + constants.DX + ":" + str(self.inconsistent_date))))
-                self.results_publish.append(constants.table_publish_format(self.analysis_name, self.timestamp[-1], (constants.ECON2 + constants.DX), self.inconsistent_date))
-                self.clear_data()
-                return
-            self.not_economizing_when_needed()
-
-    def economizer_conditions(self, cooling_call, econ_condition, cur_time):
+    def economizing_check(self, cooling_call, econ_condition, cur_time):
```
Comment (craigpnnl) [inline pnnl/EconomizerRCxAgent/economizer/diagnostics/EconCorrectlyOn.py:185]: Why is this function name used more than once in the code base?  Should we extract it into a single function/class that can be overridden with a __call__ functionn?

File: pnnl/EconomizerRCxAgent/economizer/diagnostics/EconCorrectlyOn.py [hunk 3]
```diff
@@ -176,31 +192,39 @@ def economizer_conditions(self, cooling_call, econ_condition, cur_time):
...
-                self.results_publish.append(constants.table_publish_format(self.analysis_name, cur_time, ( constants.ECON2 + constants.DX),  self.not_economizing_dict))
-                self.clear_data()
+            self.not_economizing.append(cur_time)
             return False
-        else:
-            self.not_economizing = None
+
         return True
 
+    def economizer_conditions(self, current_time):
```
Comment (craigpnnl) [inline pnnl/EconomizerRCxAgent/economizer/diagnostics/EconCorrectlyOn.py:205]: Same question as above

File: pnnl/EconomizerRCxAgent/economizer/diagnostics/ExcessOutsideAir.py [hunk 4]
```diff
@@ -146,40 +171,30 @@ def excess_ouside_air_algorithm(self, oat, rat, mat, oad, econ_condition, cur_ti
...
-
-        if elapsed_time >= self.data_window and len(self.timestamp) >= self.no_required_data:
-            if elapsed_time > self.max_dx_time:
-                _log.info(constants.table_log_format(self.analysis_name, self.timestamp[-1], (constants.ECON4 + constants.DX + ":" + str(self.inconsistent_date))))
-                self.results_publish.append(constants.table_publish_format(self.analysis_name, self.timestamp[-1], (constants.ECON4 + constants.DX), self.inconsistent_date))
-                self.clear_data()
-                return
-            self.excess_oa()
 
+    def economizer_conditions(self, current_time):
```
Comment (craigpnnl) [inline pnnl/EconomizerRCxAgent/economizer/diagnostics/ExcessOutsideAir.py:175]: Same Question

File: pnnl/EconomizerRCxAgent/economizer/diagnostics/ExcessOutsideAir.py [hunk 5]
```diff
@@ -146,40 +171,30 @@ def excess_ouside_air_algorithm(self, oat, rat, mat, oad, econ_condition, cur_ti
...
+                                               current_time,
+                                               (constants.ECON4 + constants.DX),
+                                               self.economizing_dict))
+            self.clear_data()
+            return True
+        return False
 
-    def economizer_conditions(self, econ_condition, cur_time):
-        """Check conditions to see if should be economizing
+    def economizing_check(self, econ_condition, cur_time):
```
Comment (craigpnnl) [inline pnnl/EconomizerRCxAgent/economizer/diagnostics/ExcessOutsideAir.py:188]: Same Question

**Review by craigpnnl — CHANGES_REQUESTED** [review]

File: pnnl/EconomizerRCxAgent/economizer/diagnostics/ExcessOutsideAir.py (hunk 5 above)
Comment (rlutes) [reply pnnl/EconomizerRCxAgent/economizer/diagnostics/ExcessOutsideAir.py:188]: Those functions do similar but different checks.

**Review by craigpnnl — APPROVED** [review]

**Pull request merged**